### ISSUE TITLE
ComboBox input has "none" as default value of aria-autocomplete 

### ISSUE THREAD

**Issue by dengeorge** [microsoft/fluentui#22084]
### Environment Information

- **Package version(s)**: All (Codepen is on v8)
- **Browser and OS versions**: Edge (Tested on 99.0.1150.36) [Note that this [setting](https://www.tenforums.com/tutorials/160845-enable-disable-save-fill-personal-info-microsoft-edge.html) must be enabled]

### Please provide a reproduction of the bug in a codepen:
[Overlapping Menus Issue](https://codepen.io/dengeorge/pen/qBpEZwv)

#### Actual behavior:
Edge shows autofill info for ComboBox inputs having id or aria-label starting with "name", which isn't the desired behavior for comboboxes since it will result in two overlapping menus. On investigating the attributes, [aria-autocomplete flag](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-autocomplete) seems to be initialized incorrectly for the underlying [autofill](https://github.com/microsoft/fluentui/blob/83a08f167c8d8535f4e15ab2b760bec1a1a486be/packages/react/src/components/ComboBox/ComboBox.tsx#L2344). I feel it should be "list" here rather than "none" as per the definition of the values of this attribute.

#### Expected behavior:

aria-autocomplete should be "list" in case of allowFreeform==true && autoComplete==false
When this attribute is changed, Edge does not display the autofill.

### Priorities and help requested:

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

**Comment by micahgodbolt** [comment]
![image](https://user-images.githubusercontent.com/1434956/159597316-3511277a-487a-49cc-be3d-ea869253ab24.png)
